src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java

```diff
@@ -55,11 +55,12 @@
  * </li>
  * </ul>
  * <p>
- * To configure the default check that will check {@code @param}, {@code @return},
- * {@code @throws}, {@code @deprecated}:
+ * To configure the check to validate {@code @param} tags:
  * </p>
  * <pre>
- * &lt;module name="NonEmptyAtclauseDescription"/&gt;
+ * &lt;module name="NonEmptyAtclauseDescription"&gt;
```

Comment:
Keep default config and example of execution in addition to what you suggest.

Reply:
@romani  Like this:
```
*<p>
* To configure the default check that will check {@code @param}, {@code @return},
 * {@code @throws}, {@code @deprecated}:
 * </p>
 * <pre>
 * &lt;module name="NonEmptyAtclauseDescription"/&gt;
 * </pre>
 * <p>
 * Example:
 * </p>
 * <pre>
 * class Test
 * {
 * &#47;**
 * * Violation for param "b" and at tags "deprecated", "throws".
 * * &#64;param a Some javadoc // OK
 * * &#64;param b
 * * &#64;deprecated
 * * &#64;throws Exception
 * *&#47;
 * public int method(String a, int b) throws Exception
 * {
 * return 1;
 * }
 * }
 * </pre>
 ```

Reply:
Yes

Reply:
I am on it.

Reply:
@romani Added changes you asked for.